File: apps/institution-api/Models/InstitutionWebsite.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionWebsite
    {
        public int InstitutionWebsiteId { get; set; }
        public int InstitutionId { get; set; }
        public Guid? WebsiteGuid { get; set; }
        public string WebsiteUsage { get; set; }
        public int? ListSequence { get; set; }
        public string WebsiteUrl { get; set; }
        public DateTime? WebsiteBusinessStartDate { get; set; }
        public DateTime? WebsiteBusinessEndDate { get; set; }

        public virtual Institution Institution { get; set; }
    }
}

File: libs/common/models/institution-website.ts


import { Institution } from './institution';

export class InstitutionWebsite {
    public institutionWebsiteId: number = 0;
    public institutionId: number = 0;
    public websiteGuid: string;
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public websiteBusinessStartDate: Date;
    public websiteBusinessEndDate: Date;
    public institution: Institution = new Institution();

    constructor(init? : Partial<InstitutionWebsite>) {
        
        Object.assign(this, init);
    }
}

